### DISCUSSION TITLE
NOCODB Post Rest Api with relational rows

### DISCUSSION THREAD

**Discussion by FahadYameen** [nocodb/nocodb#3829] [Q&A]
Hi,
I am trying to create rows using nocodb post api, its working but the column which are actually the lookup columns are not filling. I am passing the data in body but there is nothing filled in that row of lookup columns.
Is there any way to post the relational column data?

**Comment by pranavxc** [comment]
For adding LTAR data you need to call an additional API once the record is created.

For API details - https://apis.nocodb.com/#tag/DB-table-row/operation/db-table-row-nested-add
